include/deal.II/base/qprojector.h

```diff
@@ -444,6 +454,17 @@ class QProjector
          const bool                face_rotation,
          const unsigned int        n_quadrature_points);
 
+    /**
+     * TODO.
```

Comment:
Do you still expect changes here in terms of interface that motivates to postpone the documentation? If not, I suggest to fill it now because it gets forgotten otherwise. It's mostly copy-paste from the previous function anyway.